Error on latest nightly

```
Compiling tide-core v0.2.0 (https://github.com/rustasync/tide?rev=dd0bcfe#dd0bcfe9)
error[E0557]: feature has been removed
 -->~/.cargo/git/checkouts/tide-d8df193c9a60972c/dd0bcfe/tide-core/src/lib.rs:3:25
  |
3 | #![feature(async_await, existential_type)]
  |                         ^^^^^^^^^^^^^^^^
  |
note: removed in favor of `#![feature(type_alias_impl_trait)]`
```

rustc 1.38.0-nightly (2d1a551e1 2019-08-08)

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,5 @@
 language: rust
-rust: nightly-2019-07-08
+rust: nightly-2019-07-29
 
 before_script:
 - >

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,5 @@
 language: rust
-rust: nightly-2019-07-29
+rust: nightly-2019-07-31
 
 before_script:
 - >

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -31,7 +31,7 @@ http-service = "0.3.0"
 http-service-hyper = { version = "0.4.0", optional = true }
 # Routing
 fnv = "1.0.6"
-route-recognizer = "0.1.12"
+route-recognizer = "0.1.13"
 # Tide components
 tide-cookies = { path = "./tide-cookies", optional = true, default-features = false }
 tide-cors = { path = "./tide-cors", optional = true, default-features = false }

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -73,3 +73,7 @@ members = [
     "tide-querystring",
     "tide-slog",
 ]
+
+[patch.crates-io.runtime]
+git = "https://github.com/rustasync/runtime"
+rev = "f5ac4fb"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -4,7 +4,7 @@
 
 #![cfg_attr(any(feature = "nightly", test), feature(external_doc))]
 #![cfg_attr(feature = "nightly", doc(include = "../README.md"))]
-#![feature(async_await, existential_type)]
+#![feature(async_await)]
 #![warn(
     nonstandard_style,
     rust_2018_idioms,

--- a/tide-core/Cargo.toml
+++ b/tide-core/Cargo.toml
@@ -16,7 +16,7 @@ http = "0.1"
 http-service = "0.3.0"
 serde = "1.0.91"
 serde_json = "1.0.39"
-route-recognizer = "0.1.12"
+route-recognizer = "0.1.13"
 
 [dev-dependencies]
 tide = { path = "../", default-features = false }

--- a/tide-core/src/lib.rs
+++ b/tide-core/src/lib.rs
@@ -1,6 +1,6 @@
 //! Core types and traits from Tide
 
-#![feature(async_await, existential_type)]
+#![feature(async_await)]
 #![warn(
     nonstandard_style,
     rust_2018_idioms,

